Tests/test_PDB.py

```diff
@@ -1332,6 +1332,25 @@ def test_NACCESS_asa_file(self):
             naccess = process_asa_data(asa)
         self.assertEqual(len(naccess), 524)
 
+class TestSelf(unittest.TestCase):
+    
+    def test_example(self):
+        from Bio.PDB.PDBParser import PDBParser
+        import sys
+        p = PDBParser(PERMISSIVE=True)
+        s = p.get_structure("test", sys.argv[1])
+        io = PDBIO()
+        io.set_structure(s)
+        io.save("out1.pdb")
```

Comment:
See the ``tempfile`` module for how to generate a temporary file - and then after this test please delete it.